DifferenceGenerator/src/main/kotlin/algorithms/Gotoh.kt

```diff
@@ -104,23 +107,27 @@ class Gotoh<T>(
 
         // now, we need to trace back through the matrices to retrieve the optimal alignment
         // for that, we start at the end of the alignment, store the alignment elements and reverse
-        // them afterwards
+        // them afterward
         // to get the final alignment sequence
 
-        val traceback: ArrayList<AlignmentElement> = ArrayList<AlignmentElement>()
+        val traceback: ArrayList<AlignmentElement> = ArrayList()
 
         var i: Int = n
         var j: Int = m
 
         // variable to store the last alignment "action"
         var origin =
             (
-                if (finalScore == score[n][m]) {
-                    AlignmentElement.MATCH
-                } else if (finalScore == gapA[n][m]) {
-                    AlignmentElement.DELETION
-                } else {
-                    AlignmentElement.INSERTION
+                when (finalScore) {
+                    score[n][m] -> {
+                        AlignmentElement.MATCH
+                    }
+                    gapA[n][m] -> {
+                        AlignmentElement.DELETION
+                    }
+                    else -> {
+                        AlignmentElement.INSERTION
+                    }
                 }
             )
```

Comment:
This line is marked by ktlint in my IDE. The parenthesis should apprently be idented like the one in the line above. (Kinda strange because it looks right to me).